components/clarinet-cli/src/generate/project.rs

```diff
@@ -425,11 +425,11 @@ btc_address = "mvZtbibDAAA3WLpY7zXXFqRa3T4XSknBX7"
   "license": "ISC",
   "dependencies": {{
     "@hirosystems/clarinet-sdk": "^2.3.2",
-    "@stacks/transactions": "^6.12.0",
+    "@stacks/transactions": "^7.0.0",
     "chokidar-cli": "^3.0.0",
     "typescript": "^5.3.3",
     "vite": "^5.1.4",
-    "vitest": "^1.3.1",
+    "vitest": "^2.1.4",
```

Comment:
vitest released a v3, you might want to jump from 1 > 3 directly as this pr is a breaking change see https://github.com/vitest-dev/vitest/releases/tag/v3.0.0 :)

Reply:
@pradel 
You can start using vitest v3 today, just make to have
- `@hirosystems/clarinet-sdk` >= 2.14.0
- `vitest-environment-clarinet` >= 2.3.0

And you'll be able to use any vitest version from 1 to 3.
We'll default to 3.0 in future versions 👍